charts/grafana/values.yaml

```diff
@@ -294,7 +295,7 @@ extraContainerVolumes: []
 ##
 persistence:
   type: pvc
-  enabled: false
+  enabled: enabled
```

Comment:
it's better to mention it in upgrade guide

Reply:
While one could do that I think this would avoid some people missing to turn PVC on and thus losing data when the Pod restarts.